Modules/Scripted/SegmentStatistics/SegmentStatistics.py

```diff
@@ -802,7 +802,7 @@ def test_SegmentStatisticsBasic(self):
             sphereSource.SetCenter(segmentGeometry[1], segmentGeometry[2], segmentGeometry[3])
             sphereSource.Update()
             uniqueSegmentID = segmentationNode.GetSegmentation().GenerateUniqueSegmentID("Test")
-            segmentationNode.AddSegmentFromClosedSurfaceRepresentation(sphereSource.GetOutput(), uniqueSegmentID)
+            segmentationNode.AddSegmentFromClosedSurfaceRepresentation(sphereSource.GetOutput(), uniqueSegmentID, None, uniqueSegmentID)
```

Comment:
It obscures the intent of the test a bit that we use segment ID as both segment name and ID. Something like this would be more clear:
```suggestion
            segmentationNode.AddSegmentFromClosedSurfaceRepresentation(sphereSource.GetOutput(), None, None, uniqueSegmentID)
```